### DISCUSSION TITLE
graphsage accuracy

### DISCUSSION THREAD

**Discussion by Chrisable** [pyg-team/pytorch_geometric#4611] [General]
I find the accuracy of some paper is more than 80% by using graphsage model on Cora dataset.However,the code of the [link](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/graph_sage_unsup.py) is just 75%.Why?How can I improve accuracy？

**Comment by rusty1s** [comment]
Note that this is for unsupervised training. I think it is easily possible to get to 80% when using supervised training of GraphSAGE.

**Reply by Chrisable** [reply]
where is the example of supervised training?Could you provide a link?

**Reply by rusty1s** [reply]
You can just swap out the GNN layers here: https://github.com/pyg-team/pytorch_geometric/blob/master/examples/gcn.py
